private/bufpkg/bufconfig/buf_yaml_file.go

```diff
@@ -74,6 +74,18 @@ type BufYAMLFile interface {
 	// All ModuleConfigs will have unique ModuleFullNames.
 	// Sorted by DirPath.
 	ModuleConfigs() []ModuleConfig
+	// DefaultLintConfig returns the DefaultLintConfig for the File.
+	//
+	// For v1 buf.yaml files, this will be the default v1 lint config.
```

Comment:
Why we wouldn't return the top-level config always for v1 buf.yaml files to match v2 behavior?

Reply:
Hmm... yeah, I suppose that is correct -- since for `v1` `buf.yaml`'s, there really only is "one top-level" config, and if that isn't set, it just returns default v1, and we should stick to that. Going to adjust the docs and the code accordingly.